src/Uno.UWP/UI/Core/ICoreWindowExtension.cs

```diff
@@ -11,5 +12,17 @@ internal interface ICoreWindowExtension
 		void ReleasePointerCapture(PointerIdentifier pointer);
 
 		void SetPointerCapture(PointerIdentifier pointer);
+
+#if UNO_SUPPORTS_NATIVEHOST
+		bool IsNativeElement(object content);
+
+		void AttachNativeElement(object owner, object content);
+
+		void DetachNativeElement(object owner, object content);
+
+		void ArrangeNativeElement(object owner, object content, Rect arrangeRect);
+
+		Size MeasureNativeElement(object owner, object content, Size size);
+#endif
```

Comment:
Wondering if we should put all of this on the same `ICoreWindowExtension`, not even sure if we should suffix this interface by `Extension` actually. I would split it multiple interface like `IPointersSTUFF` and `INativeElementSTUFF`.

Reply:
It may make sense. We may need to revisit this if additional scenarios are needed, though the interface is internal so we can change easily.